babun-core/plugins/shell-here/install.sh

```diff
@@ -0,0 +1,22 @@
+set -e -f -o pipefail
+source "/usr/local/etc/babun.instance"
+source "$babun_tools/script.sh"
+
+babun_root=$( cygpath -w "/" | sed "s#\\\cygwin##g" )
+name="Babun Shell here"
+cmd="${babun_root}\cygwin\bin\mintty.exe /bin/env CHERE_INVOKING=1 /bin/zsh.exe"
+
+keys=("HKCU\Software\Classes\Directory\Background\shell\babun"
+	"HKCU\Software\Classes\Directory\shell\babun"
+	"HKCU\Software\Classes\Drive\Background\Shell\babun"
+	"HKCU\Software\Classes\Drive\shell\babun")
+
+#start with installing chere
+pact install chere || echo "Installing 'chere' failed"
+
+#install registry keys
+for key in ${keys[*]}
+do 
+	cmd /c "reg" "add" "${key}" "/ve" "/d" "${name}" "/t" "REG_SZ" "/f" || echo "Failed adding ${key}"
+	cmd /c "reg" "add" "${key}\command" "/ve" "/d" "${cmd}" "/t" "REG_EXPAND_SZ" "/f" || echo "Failed adding ${key}"
```

Comment:
Yep, it was a good option, but there was something else that I wanted to implement for a long period of time - namely a new `hook` for plugins. If a plugin implements an exec.sh script now (similar to install.sh or start.sh), whenever `babun <plugin_name> xxx yyy` command is invoked, the execution is passed to `<plugin_name>/exec.sh` script with params `xxx yyy` script. Thus, a plugin does not have to implement it's own /usr/local/bin/xxx script to enable the user to execute some additional actions.

We've added exec.sh in shell-here plugin to allow overwriting the registry entries.
One can just execute `babun shell-here init` and the registry entries will be overwritten.
We hope you like it, since you are the author of the shell-here plugin :)